modules/idlgen/plugin/src/sbt-test/sbt-mu-idlgen/srcProtoGenFromDirs/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.1.5
```

Comment:
I think we can remove the build.properties file from scripted tests

Reply:
👍 Just cloned the most similar project. I'll do it.